tox.ini

```diff
@@ -2,38 +2,36 @@
 envlist =
     flake8,
     py27,
-    py35,
     py36,
+    py37,
     coverage,
 
 skip_missing_interpreters = False
 
 [testenv]
+usedevelop = true
 commands =
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} bootstrap
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} -n install test
-    coverage run {envbindir}/test --all {posargs:-vc}
-skip_install = true
+    zope-testrunner --test-path=src --all {posargs:-vc}
+extras = zrs
 deps =
-    setuptools==33.1.1
-    zc.buildout
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
+    zope.testrunner
 
 [testenv:coverage]
-basepython = python2.7
-deps = coverage
+basepython = python3.6
+deps =
+    zope.testrunner
+    coverage
 setenv =
-    COVERAGE_FILE=.coverage
+    COVERAGE_HOME={toxinidir}
+    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
 commands =
-    coverage erase
+    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
     coverage combine
```

Comment:
Do we need the combine here, when we only run the coverage in a single testenv?

Reply:
Yes we need it because we write coverage in different processes when running buildout. Without combining them we get about 8 % test coverage.